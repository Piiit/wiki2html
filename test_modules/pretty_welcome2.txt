{{authorA:="Lorenzo"}}
{{authorB:="Peter"}}
= This is an example generated with wiki2html=
Welcome from {{authorA}} & {{authorB}}!
==What is that?==
This is a **wiki** parser basically, where you can easily produce HTML-compliant files in a go.
==Some features==
* implements wiki syntax ([[http://en.wikipedia.org/wiki/Creole_%28markup%29|creole wiki markup]])
* produces HTML compliant output
* easy to use
* free and portable

==More information==
You can find more information about the syntax [[http://www.wikicreole.org/wiki/AllMarkup|here]]
==Is that free?==
Yes it is absolutely //free// and //open source//.
==Who wrote it?==
{{authorA}} & {{authorB}} for a Compilers course project.
==Okay now something serious, lets test something more==
Printing an undefined variable -> {{undefined}} <- no result as in PHP, just a warning when compiling...

