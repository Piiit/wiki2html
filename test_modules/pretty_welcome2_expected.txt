<!doctype html>
<html>
<head>
<title>wiki2html: no title</title>
</head>
<body>


<h1>This is an example generated with wiki2html</h1>
Welcome from Lorenzo & Peter!
<h2>What is that?</h2>
This is a <b>wiki</b> parser basically, where you can easily produce HTML-compliant files in a go.
<h2>Some features</h2>
<ul>
<li>implements wiki syntax (<a href='http://en.wikipedia.org/wiki/Creole_%28markup%29'>creole wiki markup</a>)</li>
<li>produces HTML compliant output</li>
<li>easy to use</li>
<li>free and portable</li>
</ul>

<h2>More information</h2>
You can find more information about the syntax <a href='http://www.wikicreole.org/wiki/AllMarkup'>here</a>
<h2>Is that free?</h2>
Yes it is absolutely <i>free</i> and <i>open source</i>.
<h2>Who wrote it?</h2>
Lorenzo & Peter for a Compilers course project.
<h2>Okay now something serious, lets test something more</h2>
Printing an undefined variable ->  <- no result as in PHP, just a warning when compiling...

</body>
</html>